src/components/Volume/index.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actions from '../../actions/index';
+import * as toggleTypes from '../../constants/toggleTypes';
+import Slider from 'react-rangeslider';
+
+class VolumeSlider extends React.Component {
```

Comment:
Why did you choose to have a stateful component in the first place? In VolumeSlider you never use this.state to retrieve the state. Furthermore in handleChange you set the internals component state, but you never use it. Instead of calling this.handleChange you could simply call onChange={onChangeVolume}. After that you can convert the component to a stateless functional component :)